pulsar-broker-common/src/main/java/org/apache/pulsar/broker/resources/LoadBalanceResources.java

```diff
@@ -69,4 +73,22 @@ private String getBundleDataPath(final String bundle) {
             return BUNDLE_DATA_BASE_PATH + "/" + bundle;
         }
     }
+
+    public static class BrokerTimeAverageDataResources extends BaseResources<TimeAverageBrokerData> {
+        public BrokerTimeAverageDataResources(MetadataStore store, int operationTimeoutSec) {
+            super(store, TimeAverageBrokerData.class, operationTimeoutSec);
+        }
+
+        public CompletableFuture<Void> updateTimeAverageBrokerData(String broker, TimeAverageBrokerData data) {
```

Comment:
maybe we can change args broker to `brokerLookupAddress` or add javadoc to call the api caller how to pass args broker here.

Reply:
En.. I think that both `brokerLookupAddress` and `broker` is OK.  We can see that other place  also name the arg as `broker`, e.g.:
https://github.com/apache/pulsar/blob/a10564d875f42c32fda0178cfe7ad976d26fbfb1/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ModularLoadManagerImpl.java#L1179-L1187